lms/static/coffee/spec/instructor_dashboard/student_admin_spec.coffee

```diff
@@ -0,0 +1,166 @@
+describe 'StudentAdmin', ->
+  studentadmin = {}
+  beforeEach =>
+    loadFixtures 'coffee/fixtures/student_admin.html'
+    # have initialize InstructorDashboard.util.PendingInstructorTasks
+    # since it is user in construct of StudentAdmin
+    window.InstructorDashboard = {}
+    window.InstructorDashboard.util =
+      std_ajax_err: std_ajax_err
+      PendingInstructorTasks: PendingInstructorTasks
+      create_task_list_table: create_task_list_table
+
+    studentadmin = new StudentAdmin $('#student_admin')
+  it 'expects student admin container to be visible', ->
+    expect($('#student_admin')).toBeVisible()
+
+  describe 'EntranceExam', =>
+    it 'Expects student identified input to be visible', =>
+      expect(studentadmin.$field_entrance_exam_student_select_grade).toBeVisible()
+
+    it 'binds to the btn_reset_entrance_exam_attempts on click event', =>
+      expect(studentadmin.$btn_reset_entrance_exam_attempts).toHandle 'click'
+
+    it 'binds to the btn_delete_entrance_exam_state on click event', =>
+      expect(studentadmin.$btn_delete_entrance_exam_state).toHandle 'click'
+
+    it 'binds to the btn_rescore_entrance_exam on click event', =>
+      expect(studentadmin.$btn_rescore_entrance_exam).toHandle 'click'
+
+    it 'binds to the $btn_entrance_exam_task_history on click event', =>
+      expect(studentadmin.$btn_entrance_exam_task_history).toHandle 'click'
+
+    it 'binds to the btn_rescore_entrance_exam on click event', =>
+      expect(studentadmin.$btn_rescore_entrance_exam).toHandle 'click'
+
+    it 'Expects student task history table to be visible', =>
+      expect(studentadmin.$table_entrance_exam_task_history).toBeVisible()
+
+
+    it 'binds reset entrance exam ajax call and the result will be success', =>
+      studentadmin.$btn_reset_entrance_exam_attempts.click()
+      expect(studentadmin.$request_response_error_ee.text()).toEqual(
+        gettext("Please enter a student email address or username.")
+      )
+
+      spyOn($, "ajax").andCallFake((params) =>
```

Comment:
For example, see these instructor dashboard tests:

https://github.com/edx/edx-platform/blob/master/lms/static/js/spec/groups/views/cohorts_spec.js#L591